src/audio/drc/drc_math_generic.c

```diff
@@ -225,12 +225,11 @@ inline int32_t drc_inv_fixed(int32_t x, int32_t precision_x, int32_t precision_y
  */
 inline int32_t drc_sin_fixed(int32_t x)
 {
-	const int32_t PI_OVER_TWO = Q_CONVERT_FLOAT(1.57079632679489661923f, 30);
+	const int32_t PI_OVER_TWO = Q_CONVERT_FLOAT(1.57079632679489661923, 30);
```

Comment:
These will only be inline if they are used in the same C file. @ShriramShastry can you do an incremental update that takes all these small inline maths function and moves them to the maths headers (and changes them to static inline).
This will save us a function call on all our small maths ops.

Reply:
Thanks Liam. I have a new PR https://github.com/thesofproject/sof/pull/4218 to include your inputs.